jest/jest-setup-after-env.ts

```diff
@@ -34,11 +32,8 @@ const $ = {
     development($),
     testing($),
   )
-
-  /**
-  * Adds the application’s router to vue test utils. This way tests don’t have to set-up a new router instance on their own.
-  */
-  config.global.plugins.push(get($.router))
+  // initializes vue-test-utils with any global components and/or plugins etc
+  get($.app)
```

Comment:
This now shares the same `$.components` and `$.plugins` tokens as the main application, i.e. whatever you install in the application, also gets installed for the tests. You don't have to add the things in two places anymore.